### DISCUSSION TITLE
请求添加 kmod_inet_diag 支持

### DISCUSSION THREAD

**Discussion by ghost** [Boos4721/openwrt#100] [Ideas/你的好主意]
我在尝试安装最新的openclash时遇到 kmod_inet_diag 缺失这个问题，搜索后发现需要在 package/kernel/linux/modules/netsupport.mk 中添加相关代码  
我自己尝试fork并添加后能够顺利编译暂时没有遇到问题  
但是由于lede和官方库的pr者并未向大佬的库pr，我不知道直接向大佬的库里pr别人的代码是否妥当，所以新建此 discussion 希望了解大佬的看法

相关代码:  
Ref: openwrt/openwrt#9431 coolsnowwolf/lede#9066

**Comment by ghost** [comment]


**Reply by ghost** [reply]
在Boos大没有更新前，你可以先用我fork的库 https://github.com/ishkong/openwrt 只比Boos大的库多了kmod_inet_diag

**Comment by ghost** [comment]
提供一个自己编译的ipk，有需要可以自取

https://github.com/ishkong/AX6-AX3600-openwrt-self-use/releases/download/2022.04.04-1505/kmod-inet-diag_5.10.109-1_aarch64_cortex-a53.ipk

**Comment by Boos4721** [comment]
**ADDED**

**Reply by ghost** [reply]
感谢
